bigquery/bigquerydatatransfer/snippets/src/test/java/com/example/bigquerydatatransfer/CreateAzureBlobStorageTransferIT.java

```diff
@@ -0,0 +1,109 @@
+  private static final String ID = UUID.randomUUID().toString().substring(0, 8);
+  private BigQuery bigquery;
+  private ByteArrayOutputStream bout;
+  private String name;
+  private String displayName;
+  private String datasetName;
+  private String tableName;
+  private PrintStream out;
+  private PrintStream originalPrintStream;
+
+  private static final String PROJECT_ID = requireEnvVar("GOOGLE_CLOUD_PROJECT");
+  private static final String DATASET_ID = requireEnvVar("DATASET_ID");
+  private static final String TABLE_NAME = requireEnvVar("TABLE_NAME");
+  private static final String STORAGE_ACCOUNT = requireEnvVar("STORAGE_ACCOUNT");
+  private static final String CONTAINER = requireEnvVar("CONTAINER");
+  private static final String DATA_PATH = requireEnvVar("DATA_PATH");
+  private static final String SAS_TOKEN = requireEnvVar("SAS_TOKEN");
+
+  private static String requireEnvVar(String varName) {
+    String value = System.getenv(varName);
+    assertNotNull(
+        "Environment variable " + varName + " is required to perform these tests.",
+        System.getenv(varName));
+    return value;
+  }
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    originalPrintStream = System.out;
+    System.setOut(out);
+  }
+
+  @After
+  public void tearDown() throws IOException {
+    System.out.flush();
+    System.setOut(originalPrintStream);
+    LOG.log(Level.INFO, bout.toString());
+  }
+
+  @Test
+  public void testCreateAzureBlobStorageTransfer() throws IOException {
+    Map<String, Value> params = new HashMap<>();
+    params.put(
+        "destination_table_name_template", Value.newBuilder().setStringValue(TABLE_NAME).build());
+    params.put("storage_account", Value.newBuilder().setStringValue(STORAGE_ACCOUNT).build());
+    params.put("container", Value.newBuilder().setStringValue(CONTAINER).build());
+    params.put("data_path", Value.newBuilder().setStringValue(DATA_PATH).build());
+    params.put("sas_token", Value.newBuilder().setStringValue(SAS_TOKEN).build());
+    params.put("file_format", Value.newBuilder().setStringValue("CSV").build());
+    params.put("field_delimiter", Value.newBuilder().setStringValue(",").build());
+    params.put("skip_leading_rows", Value.newBuilder().setStringValue("1").build());
+
+    TransferConfig transferConfig =
+        TransferConfig.newBuilder()
+            .setDestinationDatasetId(DATASET_ID)
+            .setDisplayName("Azure Blob Storage Transfer Code Example Test")
+            .setDataSourceId("azure_blob_storage")
+            .setParams(Struct.newBuilder().putAllFields(params).build())
+            .setSchedule("every 24 hours")
+            .build();
+    CreateAzureBlobStorageTransfer.createAzureBlobStorageTransfer(PROJECT_ID, transferConfig);
+    String result = bout.toString();
+    name = result.substring(result.indexOf(":") + 1, result.length() - 1);
```

Comment:
Should the "name" be included as part of the assertion statement?

Reply:
This var is left-over from copy-paste code. I removed it.